src/components/dialog-full-screen/dialog-full-screen-test.stories.tsx

```diff
@@ -141,6 +143,39 @@ Nested.parameters = {
   },
 };
 
+export const WithFullWidthHeader: StoryType = {
+  render: (args) => {
+    const { children, ...rest } = args;
+    return <DialogFullScreen {...rest}>{children}</DialogFullScreen>;
+  },
+  args: {
+    children: "Content",
+    open: true,
+    title: (
+      <Box maxWidth="750px" width="100%" data-testid="test">
+        <StepFlow
+          category="category"
+          title="title"
+          currentStep={1}
+          totalSteps={6}
+          showProgressIndicator
+        />
+      </Box>
+    ),
+    showCloseIcon: false,
+    onCancel: () => {},
+    hasFullWidthHeader: true,
```

Comment:
**question**: do we necessarily need a prop for controlling this, could we potentially make the header full-width by default?

Reply:
The optional prop has been removed, and the container will be full width by default.